Mathlib/Data/Rat/Cast/Defs.lean

```diff
@@ -48,6 +48,10 @@ theorem cast_natCast (n : ℕ) : ((n : ℚ) : α) = n := by
   rw [← Int.cast_ofNat, cast_intCast, Int.cast_ofNat]
 #align rat.cast_coe_nat Rat.cast_natCast
 
+-- 2024-03-21
+@[deprecated] alias cast_coe_int := cast_intCast
+@[deprecated] alias cast_coe_nat := cast_natCast
+
```

Comment:
Please split these out to another PR; the longer these are not in master, the less point there is in deprecating them. (For other reviewers: the rename already happened in another PR)

Reply:
#11636